addons/account/models/account_tax.py

```diff
@@ -1304,19 +1304,21 @@ class AccountTaxRepartitionLine(models.Model):
         help="The order in which distribution lines are displayed and matched. For refunds to work properly, invoice distribution lines should be arranged in the same order as the credit note distribution lines they correspond to.")
     use_in_tax_closing = fields.Boolean(string="Tax Closing Entry", default=True)
 
+    tag_ids_domain = fields.Binary(string="Dynamic domain used for the tag that can be set on tax", compute="_compute_tag_ids_domain")
+
+    @api.depends("tax_id", "company_id.multi_vat_foreign_country_ids")
```

Comment:
```suggestion
    @api.depends("tax_id", "company_id.multi_vat_foreign_country_ids", "company_id.account_fiscal_country_id")
```
Do you need `tax_id` in the depends, you don't seem to use it ?